### PR TITLE
feat: author clearAssets migration in generate related/assets scripts

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Robots are getting better by the day, one less manual step for engineering.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

- contributes to https://github.com/shapeshift/web/issues/6383

> What protocols, transaction types or contract interactions might be affected by this PR?

### Engineering

- Run `yarn codemod:clear-assets-migration` and ensure this adds a `clearAssets` migration
- Apply the monkey patch below, and run `yarn generate:related-asset-index`, ensuring this also adds a `clearAssets` migration

```diff
diff --git a/scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts b/scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts
index fe11012ac9..2a2946ad38 100644
--- a/scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts
+++ b/scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts
@@ -204,6 +204,7 @@ const getRelatedAssetIds = async (
   return { relatedAssetIds, relatedAssetKey }
 }
 
+// @ts-ignore
 const processRelatedAssetIds = async (
   assetId: AssetId,
   assetData: AssetsById,
@@ -248,6 +249,9 @@ const processRelatedAssetIds = async (
   return
 }
 
+export const sleep = async (intervalMs: number) =>
+  await new Promise(resolve => setTimeout(resolve, intervalMs))
+
 export const generateRelatedAssetIndex = async () => {
   console.log('generateRelatedAssetIndex() starting')
 
@@ -273,10 +277,11 @@ export const generateRelatedAssetIndex = async () => {
     drainPerInterval: 25, // Adjusted drain rate to replenish at a sustainable pace
     intervalMs: 2000,
   })
-  for (const batch of chunkArray(Object.keys(generatedAssetData), BATCH_SIZE)) {
+  for (const batch of chunkArray([1, 2, 3], BATCH_SIZE)) {
     await Promise.all(
+      // @ts-ignore
       batch.map(async assetId => {
-        await processRelatedAssetIds(assetId, assetDataWithRelatedAssetKeys, relatedAssetIndex)
+        await sleep(20)
         await throttle()
         return
       }),
```
### Operations

- None

### CODE REVIEW

**Comment by gomesalexandre** [comment]
* **#6588** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6588?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "clean": "yarn clean:packages && yarn clean:web && rm -rf node_modules coverage tsconfig.tsbuildinfo",
     "clean:web": "rm -rf build tsconfig.web.tsbuildinfo",
     "clean:packages": "yarn workspaces foreach -ptiv --exclude @shapeshiftoss/web run clean && rm -rf tsconfig.packages.tsbuildinfo",
+    "codemod:clear-assets-migration": "jscodeshift -t src/state/migrations/transformClearAssets.ts --parser=tsx --extensions=ts,tsx src/state/migrations/index.ts",
```
Comment (kaladinlight) [inline package.json:18]: can be more explicit given the file being transformed
```suggestion
    "codemod:clear-assets-migration": "jscodeshift -t src/state/migrations/transformClearAssets.ts --parser=ts --extensions=ts src/state/migrations/index.ts",
```

File: scripts/generateAssetData/generateAssetData.ts [hunk 2]
```diff
@@ -28,6 +30,8 @@ import { overrideAssets } from './overrides'
 import * as polygon from './polygon'
 import { filterOutBlacklistedAssets } from './utils'
 
+const clearAssetsCodemodCommand = 'yarn run codemod:clear-assets-migration'
```
Comment (kaladinlight) [inline scripts/generateAssetData/generateAssetData.ts:33]: i would get rid of pify and the manual exec and just chain the yarn commands in package.json

Comment (gomesalexandre) [reply scripts/generateAssetData/generateAssetData.ts:33]: 🧠

File: package.json (hunk 1 above)
Comment (gomesalexandre) [reply package.json]: c3db66ed7cb0e07071830dfc66f95a23b86fffd7

File: scripts/generateAssetData/generateAssetData.ts (hunk 2 above)
Comment (gomesalexandre) [reply scripts/generateAssetData/generateAssetData.ts]: ac6ee8eb9f4c4d2341b4e42cd802ec65ac507f8a

File: package.json (hunk 1 above)
Comment (gomesalexandre) [reply package.json]: And confirmed the ts parser is happy as well: a4e21cb7e27db6c1d589ea07c9b2d5a14bebf056

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**